test/addons/ocm-cluster/start

```diff
@@ -97,6 +98,19 @@ def join_cluster(cluster, hub):
 
 def accept_cluster(cluster, hub):
     print("Accepting cluster")
+
+    managed_clusters = clusteradm.get("clusters", output="yaml", context=hub)
+
+    managed_clusters = managed_clusters.split("---")
+    for mc in managed_clusters:
+        managed_cluster = yaml.safe_load(mc)
+        metadata_name = managed_cluster["metadata"]["name"]
+        hub_accepts_client = managed_cluster["spec"]["hubAcceptsClient"]
+
+        if metadata_name == cluster and hub_accepts_client:
+            print(f"Cluster '{cluster}' already accepted")
+            return
```

Comment:
Since this is a temporary hack until this is fixed in clusteradm, it will be better to put this code in a helper like is_accepted(), something like:

    # TODO: Remove when {link to clusteradm issue} is fixed.
    if is_accepted(cluster, hub):
        return